### DISCUSSION TITLE
How to configure/create alert event with id 0xA1E77000?

### DISCUSSION THREAD

**Discussion by MaceWindu** [datalust/seq-tickets#1875] [Help] [answered]
I'm creating alerting Seq App and have some troubles with testing of different event types. As I can see from sources of other alerting apps, there are 3 types of alert events:
- [x] log event could be sent to alerting application directly
- [x] 0xA1E77001 event generated by alert
- [ ] 0xA1E77000 event for dashboard/chart-based alerts

First two alerts are easy to generate, but I cannot find the way to configure and generate alert of last type. When I go to dashboard and create alert from chart event I get same event I can create from Alerts menu (0xA1E77001).

Is it even possible to create such event or it is some legacy from older Seq versions (I use 2023.2)?

**Comment by liammclennan** [comment, marked as answer]
Hi @MaceWindu 

The new alerts feature in [2021.3](https://blog.datalust.co/seq-2021-3-has-shipped/) replaced the previous dashboard alerts.

**Reply by MaceWindu** [reply]
Thanks!
